cypress/support/pages/teams/pageCommonTeams.js

```diff
@@ -12,13 +12,69 @@ class Teams_Common {
   static #teamsOverviewNavigationButton = '[data-testid="Teams"]'
   static #messangerActivation = '[data-testid="rocketchat_checkbox"]'
   static #saveButton = '[data-testid="create_team_btn"]'
+  static #newsTabOnTeamDetail = '[data-tab="js-news"]'
+  static #createNewsButtonOnTeamDetail = '[data-testid="create_a_team"]'
+  static #threeDotManageTeam = '[id="page-title"]'
+  static #manageTeamMembersOption = '[data-testid="administrate_team_members"]'
+  static #addInternalTeamMemberButton = '[data-testid="internal_team_members"]'
+  static #selectInternalTeamMember = '[data-testid="select_team_members_add"]'
+  static #studentTableBody = '[data-testid="students_names_container"]'
+  static #confirmTeamMemberAddButton = '[data-testid="btn-submit"]'
+  static #confirmTeamMemberDeleteButton= '[data-testid="btn-submit"]'
```

Comment:
It is the same data-testid for 2 different buttons. Can the data-testid here be "more unique"?
Would be better to handle the buttons.